addons/udes_stock_refactoring/models/refactor_group_by_move_line_key.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from itertools import groupby
+
+from odoo import models, fields, _
+
+from .refactor import Refactor, get_selection
+
+
+class ByMoveLineKey(Refactor):
```

Comment:
You have `GroupByMoveKey` make these consistent :)

Reply:
Whoops, will amend. :)

Reply:
This has now been renamed. :)